src/Symfony/Component/PropertyInfo/AccessorExtractorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * Extracts read accessor and write mutator for property of a class.
+ *
+ * @author Joel Wurtz <jwurtz@jolicode.com>
+ */
+interface AccessorExtractorInterface extends ReadAccessorExtractorInterface, WriteMutatorExtractorInterface
```

Comment:
Does this interface make sense? PropertyInfo has individual interfaces for specific purposes and one common `PropertyInfoExtractorInterface` implementing all of the individual ones. Unless you have a specific reason for introducing this intermediate interface, I would remove it.